src/xenia/kernel/xboxkrnl/xboxkrnl_usbcam.cc

```diff
@@ -17,11 +17,15 @@ namespace xe {
 namespace kernel {
 namespace xboxkrnl {
 
-dword_result_t XUsbcamCreate(unknown_t unk1,  // E
-                             unknown_t unk2,  // 0x4B000
+dword_result_t XUsbcamCreate(dword_t buffer,
+                             dword_t buffer_size,  // 0x4B000 640x480?
                              lpunknown_t unk3_ptr) {
-  // 0 = success.
-  return X_ERROR_DEVICE_NOT_CONNECTED;
+  // This function should return success.
+  // It looks like it only allocates space for usbcam support.
+  // returning error code might cause games to initialize incorrectly.
+  // "Carcassonne" initalization function checks for result from this
+  // function. If value is different than 0 it ends function with nullptr.
```

Comment:
"ends function with nullptr" sounds weird, more like "dereferences a null pointer and crashes" (if you can explain what that null pointer is and how it's obtained, better explain that here too).

Reply:
(saw the disassembly, better explain what it does with nullsub)